### DISCUSSION TITLE
Is it possible to change format of date displayed in resourceTimeGridWeek and resourceTimeGridDay view?

### DISCUSSION THREAD

**Discussion by worxflowadmin** [vkurko/calendar#208] [Q&A] [answered]
Hello, 
I want to change date format of date displayed int resourceTimeGridWeek which is showing as -> Sat, 25/11 currently.
I have attached photo file with this. I want to change the format of date shown in red box in attached image.
Please let me know if that is possible or not. If possible how can I achieve that?
![Firefox_Screenshot_2023-11-25T07-11-22 688Z](https://github.com/vkurko/calendar/assets/151848885/87bc560a-fa3b-4fa0-84f6-d67a934bcf7f)

Thank you.

**Comment by vkurko** [comment, marked as answer]
Yes, you can change the format of the displayed date in this place. See [dayHeaderFormat](https://github.com/vkurko/calendar#dayheaderformat) option.

**Reply by worxflowadmin** [reply]
Thank you, that work as expected.
